SPEC.md

```diff
@@ -52,7 +52,7 @@ at all.
 Each index entry has the following fields:
 
 * Byte length of the entry name (uint32).
-* Entry name (UTF-8 string).
+* Entry name (UTF-8 string in UNIX format).
```

Comment:
UNIX or POSIX?

Reply:
I've used UNIX as POSIX also mentions remote paths (https://en.wikipedia.org/wiki/Path_(computing)#POSIX_pathname_definition) and UNIX is already mentioned in file mode. I don't have a preference for any of them.

Reply:
Another concern is that normalization affects path encoding, and in particular path comparison. That's not something this PR needs to address, but it's something to keep in mind for any storage format.

Reply:
Both calls to `Find` and `Glob` that do searches in the file index use the same normalization for input so maybe this is enough.

* Add `user\files\test.txt`
* It's normalized and saved as `user/files/test.txt`
* Call `Gob("user\\files\\*.txt")`, converted to `user/files/.txt` so the file is found

@creachadair is this what you mean?

Reply:
> @creachadair is this what you mean?

That's part of it. I also mean normalization at the Unicode level: e.g., `é` can be represented as either `LATIN SMALL LETTER E` plus `COMBINING ACCUTE ACCENT` (decomposed) or as `LATIN SMALL LETTER E WITH ACUTE` (composed). The two are equivalent but not byte-for-byte equal in UTF-8 (`\xc3\xa9` vs. `\x65\xcc\x81`).

Worse, some filesystems (notably Apple's HFS family) require a particular normalization and will _canonicalize_ paths when they are opened. So if you are storing and querying path, you need to either decode both the needle and the haystack and compare rune-by-rune, or remember which encoding you used originally so you can canonicalize the needle at query time. (The former is easy to describe but hard to do efficiently)